Documentation/gettingstarted/k8s-install-rke.rst

```diff
@@ -0,0 +1,131 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_install_rke:
+
+********************************************
+Installation using Rancher Kubernetes Engine
+********************************************
+
+The guide walks you through integrating Cilium CNI with Kubernetes clusters
+deployed using Rancher Kubernetes Engine (RKE). The guide uses:
+
+  - Rancher Kubernetes Engine v1.0.8
+  - `Helm`_ Version 3
+
+Please consult `RKE Requirements`_ and Cilium :ref:`admin_system_reqs`.
+
+.. _RKE Requirements: https://rancher.com/docs/rke/latest/en/os/
+.. _Helm: https://helm.sh
+
+Installing Rancher Kubernetes Engine
+====================================
+
+Install the RKE Binary according to the `RKE Installation Guide <https://rancher.com/docs/rke/latest/en/installation/>`_
+
+Alternatively, on macOS the binary can be installed using Homebrew:
+
+.. code:: bash
+
+  $ brew install rke
+
+If you have a previous version of RKE installed, you can update it:
+
+
+.. code:: bash
+
+  $ brew upgrade rke
+
+Prepare the Kubernetes Cluster
+==============================
+
+Prepare your hosts with a supported operating system and Docker runtime. This
+is outside the scope of this guide.
+
+Prepare a RKE Configuration
+---------------------------
+
+Once your nodes are running and have been configured with the docker runtime,
+the next step is to generate a `RKE Configuration File <https://rancher.com/docs/rke/latest/en/installation/#creating-the-cluster-configuration-file>`_
+
+You can create a new cluster.yaml file using prompts:
+
+.. code:: bash
+
+  $ rke config --name cluster.yaml
+
+You can also use the `minimal example <https://rancher.com/docs/rke/latest/en/example-yamls/#minimal-cluster-yaml-example>`_
+
+When specifying the Network Provider Type enter none.
+
+.. code-block:: yaml
+
+  {network: {plugin: none}}
```

Comment:
What does this line refer to?  

When I generate a yaml with the prompts, I see: 

```
network:
  plugin: none
  options: {}
  mtu: 0
  node_selector: {}
  update_strategy: null
```

What are the brackets for, or is that RST formatting?

Reply:
These will render as YAML, I added the additional fields but think your comment about just generating one from prompts is easiest.